tests/test_four_c.py

```diff
@@ -275,6 +276,25 @@ def test_meshpy_locsys_condition(
     # Add locsys condition with rotation
     input_file.add(LocSysCondition(beam_set["end"], Rotation([0, 0, 1], 0.1)))
 
+    # Add line Function with function array
+
+    fun_2 = Function("SYMBOLIC_FUNCTION_OF_SPACE_TIME t")
```

Comment:
```suggestion
    # Add line Function with function array
    fun_2 = Function("SYMBOLIC_FUNCTION_OF_SPACE_TIME 2.0*t")
```
If you make the functions different, then the test will detect if they are for whatever reason switched.